content/articles/how-to-format-form-data-as-json/index.md

```diff
@@ -278,7 +266,6 @@ The sample form data (`formDataToJsonString`) object sent as JSON;
 Implementing a simple API using Nodejs and Express.js, will expose an endpoint where you will be sending the request containing the form data as JSON.
```

Comment:
This section seems too scanty. Is that all, or it's missing something? Please fix it.

Reply:
@Jethro-magaji 
My main aim in the tutorial is to demonstrate how to convert form data to JSON, the API part is only to demonstrate it actually achieves that aim(sends JSON data). Nevertheless, there are numerous articles on section.io on how to make a full-fledged API with Nodejs. If this last part is getting in the way of what I intend to do I can equally remove it, is that ok?

Reply:
@gisioraelvis If it's important, add a link to an article that would give the reader an idea of what you mean, else remove the section.

Reply:
or equally add a link to already [existing nodejs](https://www.section.io/engineering-education/restful-web-api-using-nodejs-postgressql-and-express/) articles on the site

Reply:
> or equally add a link to already [existing nodejs](https://www.section.io/engineering-education/restful-web-api-using-nodejs-postgressql-and-express/) articles on the site

YES

Reply:
@Jethro-magaji 
Hello
Have added a link to existing tutorials on the site